frontend/benefit/applicant/src/components/applications/forms/application/step5/useApplicationFormStep5.ts

```diff
@@ -55,11 +67,29 @@ const useApplicationFormStep5 = (
 
   const handleBack = (): void => handleStepChange(4);
 
+  const getEmployeeConsentAttachment = (): Attachment | null => {
+    const consentArray = application.attachments?.filter(
```

Comment:
You could use find instead of filter to make this function one-liner.

Reply:
fixed